benchmark/C/Savina/src/micro/Distributed/PingPongDistributed.lf

```diff
@@ -0,0 +1,28 @@
+/**
+ * Basic benchmark from the Savina benchmark suite that is
+ * intended to measure message-passing overhead.
+ * This version is distributed, communicating using physical connections over sockets.
+ * See [Benchmarks wiki page](https://github.com/icyphy/lingua-franca/wiki/Benchmarks).
+
+ * This is based on https://www.scala-lang.org/old/node/54
+ * See https://shamsimam.github.io/papers/2014-agere-savina.pdf.
+ * 
+ * This is a distributed version, where Ping and Pong run in
+ * separate programs and can be run on different machines.
+ * 
+ * There is no parallelism in this application, so it does not benefit from being
+ * being distributed.
+ * 
+ * These measurements are total execution time, including startup and shutdown, of
+ * all three programs.
+ * 
+ * @author Edward A. Lee
+ */
+target C;
+import Ping, Pong from "../PingPong.lf"
```

Comment:
We should think about naming conventions for distributed savina benchmarks and also find a way to integrate this with the benchmark runner. But I guess we can do this further down the road when we have a few more benchmarks.

Reply:
Maybe down the line, we can augment the benchmark runner to automatically run the C benchmarks as federated.